### PR TITLE
feat: generate links for the sidebar from contentful

### PR DESCRIPTION
**Pull request by gui-santos**
# Purpose of PR

This will start generating a json file with the links that come from Contentful so we can add them at the sidebar automagically, see this (I've wrote this "Adding icons" page recently in Contentful):

Screenshot 2022-04-22 at 16 55 24

I also added the "What’s new" link to the Introduction section

Screenshot 2022-04-22 at 16 55 07

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/main/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/website/utils/contentfulSidebarLinks.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (denkristoffer) [inline packages/website/utils/contentfulSidebarLinks.json:1]: Should this file be gitignored then?

**Comment by denkristoffer** [comment]
Awesome!!

Am I understanding this right that you want to allow links from both Contentful and a local file? If so, could you explain the rationale of not going with just one of them?

File: packages/website/utils/contentfulSidebarLinks.json (hunk 1 above)
Comment (gui-santos) [reply packages/website/utils/contentfulSidebarLinks.json:1]: I didn't ignore it because the other one was not ignored

Should I ignore both of them?

**Comment by gui-santos** [comment]
> Awesome!!
> 
> Am I understanding this right that you want to allow links from both Contentful and a local file? If so, could you explain the rationale of not going with just one of them?

@denkristoffer 
We have pages that come from Contentful and pages that come from mdx files in the repo, so we need to extract sidebar links from both sources

File: packages/website/utils/contentfulSidebarLinks.json (hunk 1 above)
Comment (denkristoffer) [reply packages/website/utils/contentfulSidebarLinks.json:1]: I'd say so, it should be generated on build, right?

Comment (gui-santos) [reply packages/website/utils/contentfulSidebarLinks.json]: Screenshot 2022-04-25 at 15 13 26

we were wrong, we need these files

**Review by denkristoffer — APPROVED** [review]

**Review by Lelith — APPROVED** [review]
Looks good to me!

**Pull request merged**